libraries/authoritative_helpers.rb

```diff
@@ -17,41 +17,74 @@
 # limitations under the License.
 #
 
-def default_authoritative_config_directory
-  case node['platform_family']
-  when 'debian'
-    '/etc/powerdns'
-  when 'rhel'
-    '/etc/pdns'
-  end
-end
+module PdnsAuthoritativeResource
+  module Helpers
+    def systemd_name(name = nil)
+      "pdns@#{name}"
+    end
 
-def default_authoritative_run_user
-  'pdns'
-end
+    def sysvinit_name(name = nil)
```

Comment:
In our company, we are not supposed to create any library without name spacing in order to ease the troubleshoot and avoid any collision, that's why. Maybe the namespacing schema does not fit your need, tell how you see stuff I would be glad to change it.